### PR TITLE
Build base constructrue

### PR DESCRIPTION
**Pull request by ZQ-jhon**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -26,5 +26,6 @@
     "test/**/__snapshots__": true,
     "yarn.lock": true,
     "*.{css,sass,scss}.d.ts": true
-  }
+  },
+  "typescript.tsdk": "c:\\Users\\35003\\projects\\iopay-desktop\\node_modules\\typescript\\lib"
```
Comment (guo) [inline .vscode/settings.json:30]: ??

File: .env.tmpl [hunk 2]
```diff
@@ -0,0 +1,3 @@
+IOTEX_CORE=api.testnet.iotex.one:80
+MONGODB_URI="mongodb://localhost:27017/iotex-explorer"
```
Comment (guo) [inline .env.tmpl:2]: what is this for? still need explorer?

Comment (ZQ-jhon) [reply .env.tmpl:2]: Just copy it only, will modify it if we need.

File: .vscode/settings.json (hunk 1 above)
Comment (ZQ-jhon) [reply .vscode/settings.json:30]: Will remove it later.

**Comment by uu-z** [comment]
do some update and make a new pull request  https://github.com/iotexproject/iopay-desktop/pull/3

**Pull request closed without merging**